src/Microsoft.AspNet.Server.Kestrel/Http/FrameRequestStream.cs

```diff
@@ -135,7 +160,7 @@ private void ValidateState()
             {
                 case StreamState.Open:
                     return;
-                case StreamState.Disposed:
+                case StreamState.Closed:
                     throw new ObjectDisposedException(nameof(FrameRequestStream));
```

Comment:
A few thoughts here:

1) `ObjectDisposedException` doesn't seem like the right exception to throw here since the stream can switch between the `Open` and `Closed` states multiple times.
2) Should we really throw? How about just no-oping when someone attempts to operate on a closed stream?